pages/item/_itemId/index.vue

```diff
@@ -78,24 +78,39 @@ export default Vue.extend({
         return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Backdrop`;
       }
     },
+    getEndsAtTime(ticks: number): string {
+      const ms = ticks / 10000;
```

Comment:
We're going to be using this very often, if jf-web is any indication.

To avoid confusing conversions everywhere, I think it's be nice to have a mixin like `timeUtils` with `ticksToMs` and `msToTicks` in it.

Reply:
Added a timeUtils mixin for converting from ticks to ms and back